src/core.js

```diff
@@ -212,7 +212,7 @@ jQuery.extend( {
 	noop: function() {},
 
 	isFunction: function( obj ) {
-		return jQuery.type( obj ) === "function";
+		return obj instanceof Function && typeof obj === "function";
```

Comment:
`instanceof` goes too far; it would falsely reject functions from other realms (e.g., iframes) and probably also some host-object functions. If we pursue my suggestion from https://github.com/jquery/jquery/issues/3596#issuecomment-290899122 , it should look like `typeof obj === "function" && !obj.nodeType` (`nodeType` being our standard test for DOM nodes, cf. https://contribute.jquery.org/style-guide/js/#type-checks ).